### ISSUE TITLE
[Bug]: Context menu's background gets much darker when opened in full screen mode

### ISSUE THREAD

**Issue by sky-ricardogomes** [mallowigi/material-theme-issues#329]
### Describe the bug

The context menu's background is much darker than the opened before the full screen mode was activated, blocking the view of the code.

### To Reproduce

1. Go to full screen mode
2. Open a context menu (e.g. extracting a class method)
3. Issue: The background of the context menu gets much darker than usual
4. Exit the full screen mode (The problem starts to happen in full screen mode, but it remains after the full screen mode is deactivated)
5. Open a context menu (e.g. extracting a class method)
6. Issue: The background of the context menu gets much darker than usual

The only way to fix the issue is restarting the IDE and not use the full screen mode again. 🙁

### Expected behavior

The context menu's background should stay as it was before the full screen mode was activated.

### Material Theme Version

8.6.0

### IDE

WebStorm

### System Info

```shell
WebStorm 2023.1
Build #WS-231.8109.174, built on March 28, 2023
Runtime version: 17.0.6+10-b829.5 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 24
Registry:
    editor.focus.mode.color.light=5a5d63
    ide.editor.tab.selection.animation=true
    debugger.new.tool.window.layout=true
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=5a5d63

Non-Bundled Plugins:
    in.xiv.cs (0.1)
    com.nasller.CodeGlancePro (1.6.0-2023.1)
    com.intellij.apacheConfig (231.8109.150)
    com.intellij.properties (231.8109.91)
    String Manipulation (9.7.1)
    DuplicateLines (1.0.1)
    ru.adelf.idea.dotenv (2023.1)
    io.pivotal.jasmine (1.8)
    com.chrisrm.idea.MaterialThemeUI (8.6.0)
    me.mbolotov.cypress (1.5.2)
    jones.restarteslintaction.restart-eslint-action (0.0.1)
    org.sonarlint.idea (8.1.0.65508)
    com.microsoft.vso.idea (1.162.1)
    com.mallowigi (88.0.0)
    zielu.gittoolbox (500.0.5+213)
    izhangzhihao.rainbow.brackets (2023.2.6)
```

### Logs/Screenshots

![image](https://user-images.githubusercontent.com/113380370/230120489-e5e3ba43-cde4-44c8-982d-28df5f7ee964.png)
![image](https://user-images.githubusercontent.com/113380370/230120602-31eec0c9-f4d1-4c65-9cbd-fe9a13225d08.png)

**Comment by sky-ricardogomes** [comment]
Before coming here, I opened a issue on the webstorm's yourtrack
https://youtrack.jetbrains.com/issue/IDEA-317286/Context-menus-background-gets-much-darker-when-opened-in-full-screen-mode?s=Context-menus-background-gets-much-darker-when-opened-in-full-screen-mode

But after disabling this plugin, the problem stopped happening

**Comment by mallowigi** [comment]
Not a bug. Check the documentation

**Comment by mallowigi** [comment]
https://material-theme.com/docs/configuration/features-settings/#overlays

**Comment by sky-ricardogomes** [comment]
Thanks! But I think you misunderstood the problem, the overlay works fine outside full screen mode, it just gets much darker in full screen mode, I can't find any settings that can change the darkness of the full screen overlay.

**Comment by sky-ricardogomes** [comment]
I only found a setting to disable the overlay. And since I would like to keep the overlay active, in fullscreen mode when I open a modal, the overlay is so dark that it prevents me from viewing the code behind the modal

**Comment by mallowigi** [comment]
Oh ok... weird, I'll take a look

**Comment by mallowigi** [comment]
I took a look... and couldn't reproduce. 

![Uploading CleanShot 2023-04-11 at 22.05.13@2x.png…]()

I'm not on Windows that being said... maybe this could be the culprit?

Can you tell me if the issue was always there or if it was something recent

**Comment by mallowigi** [comment]
Well, if you have JetBrains toolbox, you could download a previous version and check if it happens as well

**Comment by sky-ricardogomes** [comment]
Sorry for the delay in answering... I downgraded the webstorm to 2022.3.4 and the context menu overlay works well in this version. But I noted that in this version, the same problem happens to the settings window overlay, going to almost black.
![image](https://user-images.githubusercontent.com/113380370/232057980-df434683-13be-4467-b66e-4369a7158089.png)

**Comment by sky-ricardogomes** [comment]
In fact, in the downgraded version, the context menu overlay doesn't work. But it doesn't turn almost black like in the last version.
![image](https://user-images.githubusercontent.com/113380370/232059358-73f2e682-f6d2-4626-bcc1-eb93024d7b7f.png)

**Comment by david-idelji** [comment]
I've noticed that same issue where going into full screen mode or zen mode would make the overlay completely black. The black overlay stays at that shade even after I exited zen mode and have to restart the IDE to get it back to normal.

I'm using Windows by the way.
WS-231.8109.174, JRE 17.0.6+10-b829.5x64 JetBrains s.r.o., OS Windows 11(amd64) v10.0
Material Theme UI 8.6.3

![image](https://user-images.githubusercontent.com/49769138/234937713-a2ac5c10-f9b5-41da-9f5d-efc69cb35433.png)
